spec/features/user_deletes_review_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature "user deletes an existing beach", %{
+  As an authenticated user
+  I want to delete a review that I wrote
+  So that no one can view it
+} do
+
+  scenario "\n delete beach from its show page" do
+    user = FactoryGirl.create(:user)
+    beach = FactoryGirl.create(:beach, user: user)
```

Comment:
Just fyi, if you create a beach your factory generates a user for your beach object since you have an association in place there:

```ruby
beach = FactoryGirl.create(:beach)
```

Put a pry after that and look at `beach.user`

That way you can do something like this;

```ruby
beach = FactoryGirl.create(:beach)
user = beach.user
```